src/main/java/ua/shekhovtsov/hw10/Car.java

```diff
@@ -8,6 +8,7 @@
     private String color;
     private int fuel;
 
```

Comment:
1) getStats method should return some information about car
2) getStats should return something if starts with "get..."